tools/scripts/features/frcnn/extract_features_frcnn.py

```diff
@@ -0,0 +1,351 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+"""
+  Run with for example:
+  python3 mmf/tools/scripts/features/frcnn/extract_features_frcnn.py \
+  --model_file model.bin --config_file config.yaml --image_dir \
+  ./example_images --output_folder ./output_features
+"""
+
+import argparse
+import copy
+import glob
+import logging
+import math
+import os
+
+import numpy as np
+import torch
+
+from modeling_frcnn import GeneralizedRCNN
+from processing_image import Preprocess
+from utils import Config
```

Comment:
These probably should be full paths, `tools.scripts.features.frcnn.modeling_frcnn`, others similarly. Relative paths cause issues in pip packages.

Reply:
I just tried it and it suddenly caused a `ModuleNotFoundError: No module named 'pytorch_lightning'` error, any obvious reason for this? I will investigate

Reply:
`pip install pytorch_lightning`. I think the PL diff landed after you started this.